### ISSUE TITLE
[wg/aria] PDF Accessibility API mappings

### ISSUE THREAD

**Issue by plehegar** [w3c/strategy#461]
Investigation
* brief abstract:
   The PDF Association want to work with ARIA Working Group to create a PDF-AAM so implementors can create the correct accessibility mappings for PDF.
* what would this work do if successful?
   It will promote better standards compliance for accessibility mappings.
* who would be interested/supportive (types of people&orgs, not individuals)?
   The chairs of the ARIA group consider this a good thing. 

cc @jnurthen @spectranaut

**Comment by plehegar** [comment]
For a Recommendation track document that would be co-published by 2 organizations, we would establish an MoU to do the joint work, then recharter the Group to put the work listed in the MoU within the scope of a working group.

There are several questions that needs answering, including:
1. is this within our technical strategy?
2. what are we (the WG and W3C) trying to achieve and what are our parameters?
3. what is the other organization trying to achieve and what are their parameters?
4. Any other consideration that we ought to take into account?

Once we have all of that, assuming no red flag, we give it all to our legal team for them to come up with a draft MoU. Once both orgs are happy with the draft, W3C then need to follow [section 9. Liaisons](https://www.w3.org/2023/Process-20231103/#Liaisons) as it would be a technical agreement. The other organization would need to follow its own process to sign it as well. We may also be able to do the rechartering in parallel if we're confident about the outcome.

**Comment by iadawn** [comment]
@plehegar Thanks for this. I spoke with Duff regarding the approach to partnering on this work. I emailed the [outcome of the conversation which Duff agreed](https://lists.w3.org/Archives/Team/team-liaisons/2024Jun/0001.html) with.

PDF Association is keen to have this material published by W3C as they see this as resulting in better uptake. Given the extensive use of PDF delivery through browsers it makes sense to look at developing and publishing this material. This should benefit assistive technology in dealing with PDFs delivered through compliant browsers.

My understanding, and forgive my memory for not knowing where this came from, was that an MoU wasn't a necessity.

**Comment by plehegar** [comment]
Thank you @iadawn .

There is a proposed charter to include the PDF accessibility in the ARIA WG.

At this point, we can close this issue and track the follow-up on the charter issue.

See https://github.com/w3c/strategy/issues/466
